next.config.mjs

```diff
@@ -1,6 +1,13 @@
 import withPlaiceholder from "@plaiceholder/next"
 import { env } from "process"
 
+function getImageHostname() {
+  if (env.NODE_ENV === "test") {
+    return "**"
+  }
```

Comment:
Is this still relevant? Could be that it is, just wnated to ask.

Reply:
Added some better comments on why that is used :)